README.md

```diff
@@ -90,13 +90,19 @@ These environment variables build a label selector that pods must match in order
 
 ## Logging
 
-Pod reaper logs in JSON format using a logrus (https://github.com/sirupsen/logrus)
+Pod reaper logs in JSON format using a logrus (https://github.com/sirupsen/logrus). 
 
 - rule load: customer messages for each rule are logged when the pod-reaper is starting
```

Comment:
This reminds me -- with the rule refactor PR I have open, I really need to update this line in particular